docs/quickstart.md

```diff
@@ -13,32 +13,463 @@ You can find the latest Sedna release [here](https://github.com/kubeedge/sedna/r
+
+### 2. Project Deployment
+
+#### 2.1 (optional) create virtual env
+
+```bash
+python3.6 -m venv venv
+source venv/bin/activate
+pip3 install -U pip
+```
+
+#### 2.2 install sedna SDK
+
+```bash
+cd $SENDA_ROOT/lib
+python3.6 setup.py bdist_wheel
+pip3 install dist/sedna*.whl
+``` 
+
+#### 2.3 Prepare your machine learning model and datasets
+
+##### 2.3.1 Encapsulate an Estimators 
+
+Sedna implements several pre-made Estimators in [example](/examples), your can find them from the python scripts called `interface`.
+Sedna supports the Estimators build from popular AI frameworks, such as TensorFlow, Pytorch, PaddlePaddle, MindSpore. Also Custom estimators can be used according to our interface document. 
+All Estimators—pre-made or custom ones—are classes should encapsulate the following actions:
+- Training
+- Evaluation
+- Prediction
+- Export/load
+
+Follow [here](/lib/sedna/README.md) for more details, a [toy_example](/examples/incremental_learning/helmet_detection/training/interface.py) like:
+
+
+```python
+
+os.environ['BACKEND_TYPE'] = 'TENSORFLOW'
+
+class Estimator:
+
+    def __init__(self, **kwargs):
+        ...
+    
+    def train(self, train_data, valid_data=None, **kwargs):     
+        ...
+    
+    def evaluate(self, data, **kwargs):
+       ...
+
+    def predict(self, data, **kwargs):
+        ...
+
+    def load(self, model_url, **kwargs):
+        ...
+
+    def save(self, model_path, **kwargs):
+        ...
+
+    def get_weights(self):
+        ...
+
+    def set_weights(self, weights):
+        ...
+```
+
+##### 2.3.2 Dataset prepare
+
+In incremental_learning jobs, the following files will be indispensable:
+
+- base model: tensorflow object detection Fine-tuning a model from an existing checkpoint.
+- deploy model:  tensorflow object detection model, for inference.
+- train data: images with label use for Fine-tuning model.
+- test data: video stream use for model inference.
+
+```bash
+# download models, including base model and deploy model.
+
+cd /
+wget https://kubeedge.obs.cn-north-1.myhuaweicloud.com/examples/helmet-detection/models.tar.gz
+tar -zxvf models.tar.gz
+
+# download train data
+
+cd /data/helmet_detection  # notes: files here will be monitored and used to trigger the incremental training
+wget  https://kubeedge.obs.cn-north-1.myhuaweicloud.com/examples/helmet-detection/dataset.tar.gz
+tar -zxvf dataset.tar.gz
+
+# download test data
+
+cd /incremental_learning/video/
+wget https://kubeedge.obs.cn-north-1.myhuaweicloud.com/examples/helmet-detection/video.tar.gz
+tar -zxvf video.tar.gz
+ 
+```
+
+#### 2.3.3 Scripts prepare
+
+In incremental_learning jobs, the following scripts will be indispensable:
+
+- train.py: script for model fine-tuning/training.
+- eval.py: script for model evaluate.
+- inference.py: script for data inference.
+
+You can also find demos [here](/examples/incremental_learning/helmet_detection).
+
+Some interfaces should be learn in job pipeline:
```

Comment:
The sentence has some thing wrong and hard to understand. We mean that there are APIs here?

Reply:
Here are the APIs that require the minimum necessary of developers